.travis.yml

```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 
 sbt_args: -no-colors
-script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc
```

Comment:
I mean - you don't need to have now multiple Travis envs (with different scala version); it's enough to define it in sbt project